app/models/spotlight/exhibit.rb

```diff
@@ -79,8 +80,8 @@ def solr_data
       end
     end
 
-    def reindex_later
-      Spotlight::ReindexJob.perform_later(self)
+    def reindex_later(user = nil)
```

Comment:
would you be ok with checking the usage in the view and having a view test that exercises a `user`-less log entry?  i think that'd be my slight preference, since a rake task (`import`) and and `ExhibitsController#process_import` both call `Exhibit#reindex_later` without a user.